### ISSUE TITLE
plugin not working since update 4.0

### ISSUE THREAD

**Issue by sarahfinucane** [rodi01/RenameIt#109]

**Comment by sarahfinucane** [comment]
running the plugin produces no result since the 4.0 update

**Comment by rodi01** [comment]
What’s your sketch version? Have you tried restarting Sketch?

**Comment by quitefrank** [comment]
Same here. Running Sketch 52.6

**Comment by rodi01** [comment]
Could you please check the log and see if there is any error? 

Open the console app and under report go to: ~/Library/Logs/com.bohemiancoding.sketch3

You should see something like this, can you send me the output when you try to run the rename command?

<img width="1214" alt="image" src="https://user-images.githubusercontent.com/52461/52508299-fe274f80-2ba8-11e9-8033-db47b9ddb277.png">

Thanks

**Comment by quitefrank** [comment]
Well, I booted up my computer again today to try and recreate it, but it seems to be working fine now O_o
To give you some context, I had recently restarted from a time machine backup and upgraded from Sketch 48 to 52.6. Though I tried several times to restart the software, or try older versions of renameit, nothing worked. Anyways, now that I've restarted my computer it seems to be working correctly.
